src/main/java/frc/robot/RobotMap.java

```diff
@@ -154,9 +163,28 @@
      */
     public static final int DPAD_LEFT = 270;
 
-    //Drivetrain Misc Constants//
-    //public static final int DRIVETRAIN_ENCODER_TICKS_PER_REVOLUTION = 42;
-    //public static final int DRIVETRAIN_WHEEL_DIAMETER = 4;
+    //Drivetrain PID/Pathweaver Constants//
+      //Pathweaver
+      public static final int DRIVETRAIN_ENCODER_TICKS_PER_REVOLUTION = 4096;
+      public static final double DRIVETRAIN_WHEEL_DIAMETER = 4.0 / 12.0;
+      public static final double DRIVETRAIN_MAX_VELOCITY = 10.0;
+      //PID
+      public static final double DRIVETRAIN_P = 1.0;
```

Comment:
Can we include comments / rename these to better explain what PIDVA values are?

Reply:
Absolutely. 
As soon as i figure out what they are

Reply:
https://docs.google.com/presentation/d/1xjtQ5m3Ay4AYxS_SfloF2n_vWZnCU25aXZuu9A59xPY/pub?start=false&loop=false&delayms=3000&slide=id.g995bd5966_0_19

Reply:
Accompanying video explaining Motion Profiling https://www.youtube.com/watch?v=8319J1BEHwM